samcli/commands/local/lib/sam_function_provider.py

```diff
@@ -123,11 +124,15 @@ def _convert_sam_function_resource(name, resource_properties, layers):
 
         LOG.debug("Found Serverless function with name='%s' and CodeUri='%s'", name, codeuri)
 
+        timeout = resource_properties.get("Timeout")
+        if isinstance(timeout, str):
+            timeout = ast.literal_eval(timeout)
```

Comment:
Why ast here instead of doing an `int()` conversation?

In either case if the string can't be casted into a int, `ast.literal_eval` will throw `SyntaxError` and `int(timeout)` will throw `ValueError`. We should be catching and handling them gracefully here.

Reply:
I did an `ast.literal_eval` here  in case the value is a float (as the parameter type is `Number`). I shall catch and re-raise as `ValueError` -- thanks!

Reply:
I have made the change. It looks like now appveyor tests are failing, however, looking a the errors, I don't _believe_ it's related to my change? (Seems to be a permissions issue.)